### ISSUE TITLE
Warning in __clear_notifier_events() when using XPMEM

### ISSUE THREAD

**Issue by sungeunchoi** [ofi-cray/libfabric-cray#918]
```
libfabric:gni:mr:__clear_notifier_events():568<warn> _gnix_notifier_get_event returned error: Invalid argument
```

This shows up with fi_rdm_pingpong in the fabtests suite, but others have reported seeing similar warnings when using XPMEM.

**Comment by hppritcha** [comment]
I'm having problems reproducing this.  Do I need to run fi_rdm_pingpong in a particular way to get the warning messages.  I'm setting FI_LOG_LEVEL to warn but unable to get the warning messages.

**Comment by sungeunchoi** [comment]
Ah I didn't notice that we weren't seeing the warnings when running from the script in fabtests-cray.  I am running this using CCM mode on 2 different nodes (rather than a single node as in fabtests-cray).

**Comment by sungeunchoi** [comment]
This happens with the miniapps also.  Possibly with all of them, but I disabled XPMEM or turned off warnings when running, because it was getting out of hand.

**Comment by hppritcha** [comment]
This will probably be fixed by #941
